debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -358,8 +376,14 @@ public String getRealDatabaseName() {
         return realDatabaseName;
     }
 
-    private ZoneId retrieveTransactionTimezone() {
-        final String serverTimezoneConfig = config().getString(SERVER_TIMEZONE_PROP_NAME, SERVER_TIMEZONE.defaultValueAsString());
+    private ZoneId retrieveTransactionTimezone(boolean supportsAtTimeZone) {
+        final String serverTimezoneConfig = config().getString(SERVER_TIMEZONE_PROP_NAME);
+
+        if (!supportsAtTimeZone && serverTimezoneConfig == null) {
+            LOGGER.warn("The ' " + SERVER_TIMEZONE_PROP_NAME
+                    + "' option should be specified to avoid incorrect timestamp values in case of different timezones between the database server and this connector's JVM.");
+        }
+
         return serverTimezoneConfig == null ? ZoneId.systemDefault() : ZoneId.of(serverTimezoneConfig, ZoneId.SHORT_IDS);
```

Comment:
Because 2016 sql query always return UTC time, maybe we should use UTC ZoneId instead of systemDefault